### PR TITLE
Remove pagination hardcode

### PR DESCRIPTION
**Pull request by AlexKovalevych**
Created possibility to set number of pagination items per page and page range value in the config

### CODE REVIEW

File: DependencyInjection/FOQAlbumExtension.php [hunk 1]
```diff
@@ -39,6 +39,14 @@ class FOQAlbumExtension extends Extension
 
         $container->setAlias('foq_album.publisher.album', $config['service']['publisher']['album']);
         $container->setAlias('foq_album.adder.photo', $config['service']['adder']['photo']);
+
+        if ($container->hasDefinition('foq_album.provider.album') || $container->hasDefinition('foq_album.provider.photo')) {
```
Comment (ornicar) [inline DependencyInjection/FOQAlbumExtension.php]: this `if` statement is not needed. Definitions will always be available.

File: DependencyInjection/FOQAlbumExtension.php [hunk 2]
```diff
@@ -39,6 +39,14 @@ class FOQAlbumExtension extends Extension
...
+            foreach (array('item_count_per_page', 'page_range') as $key) {
+                if (isset($config['pagination'][$key])) {
```
Comment (ornicar) [inline DependencyInjection/FOQAlbumExtension.php]: this `if` statement is not needed. The configuration will always be set, even if not in `config.yml`.

File: Resources/config/provider.xml [hunk 3]
```diff
@@ -7,6 +7,9 @@
     <parameters>
         <parameter key="foq_album.provider.album.class">FOQ\AlbumBundle\Provider\AlbumProvider</parameter>
         <parameter key="foq_album.provider.photo.class">FOQ\AlbumBundle\Provider\PhotoProvider</parameter>
+
+        <parameter key="foq_album.provider.item_count_per_page">10</parameter>
+        <parameter key="foq_album.provider.page_range">5</parameter>
```
Comment (ornicar) [inline Resources/config/provider.xml]: These lines should be removed. Default values are set in the Configuration class. Parameters are set in the Extension class.

**Comment by ornicar** [comment]
That's needed and nicely done, congrats. It would be even more awesome if we were able to set different values for album provider and photo provider. Care to do that?

**Comment by AlexKovalevych** [comment]
Thank you. Thats a good idea, i will implement functionality about different values for albums and photos a little bit later.

**Pull request merged**